library/Jira/Web/Table/IssuesTable.php

```diff
@@ -46,7 +46,7 @@ protected function assemble()
                     $helper->anonymize($issue->fields->summary),
                     Html::tag(
                         'span',
-                        ['class' => 'small', 'style' => 'display: block'],
+                        ['class' => 'small'],
```

Comment:
I don't like `small`. Any chance to give that class a meaning?

Reply:
I have change the class name to `issue-description` as it is used for Jira issue description field.